scss-test.scss

```diff
@@ -0,0 +1,16 @@
+.unsorted {
+  color: #FFFFFF;
+  background: red;
+  .inception-1 {
+    color: #DDD;
```

Comment:
Color literals like `#DDD` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#DDD` should be written as `#ddd`